### PR TITLE
Feature/implement find all financial transactions

### PR DESCRIPTION
**Pull request by DawidStuzynski**

### CODE REVIEW

File: src/main/java/pl/byczazagroda/trackexpensesappbackend/controller/FinancialTransactionController.java [hunk 1]
```diff
@@ -3,23 +3,27 @@
 import lombok.RequiredArgsConstructor;
 import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
-import org.springframework.web.bind.annotation.GetMapping;
```
Comment (odzio33) [inline src/main/java/pl/byczazagroda/trackexpensesappbackend/controller/FinancialTransactionController.java:6]: the wildcard import approach has advantages, but its usage can cause problems in certain situations

please turn off Wildcard imports (https://intellij-support.jetbrains.com/hc/en-us/community/posts/206203659-Turn-off-Wildcard-imports-)

Comment (DawidStuzynski) [reply src/main/java/pl/byczazagroda/trackexpensesappbackend/controller/FinancialTransactionController.java:6]: Okay, done

**Review by odzio33 — APPROVED** [review]
Looks good for me

**Review by dawciobiel — APPROVED** [review]
Looks good for me

**Review by dawciobiel — APPROVED** [review]
Looks good for me

**Pull request merged**